tmt/steps/provision/__init__.py

```diff
@@ -590,6 +610,15 @@ def _ssh_command(self, join: bool = False) -> Union[str, List[str]]:
 
             return command + ssh_options
 
+    @classmethod
+    def options(cls, how: Optional[str] = None) -> List[tmt.options.ClickOptionDecoratorType]:
+        """ Prepare command line options related to SSH-capable guests """
+        return super().options() + [
```

Comment:
Shall we include the original `how` as well?

```suggestion
        return super().options(how=how) + [
```

As for now `Guest.options()` just provides an empty list, but who knows... in the future there might be more?

Reply:
Fixed in 802bac59.